cockroachcloud/troubleshooting-page.md

```diff
@@ -38,12 +38,6 @@ Error: x509: certificate signed by unknown authority
 Failed running "sql"
 ~~~
 
-There are two possible causes of this error: incorrect [routing ID](../{{site.versions["stable"]}}/connect-to-the-database.html#connection-parameters) in the connection string, and CA certificate conflicts in the `cockroach` certificate search path.
-
-<h4>Solution: incorrect routing ID in the connection string</h4>
-
-Check if you are using the right routing ID in the [connection method](connect-to-your-cluster.html#step-3-connect-to-your-cluster). You can find your routing ID in the {{ site.data.products.db }} Console by navigating to **Cluster Overview** > **Connect** > **Select option/language** and select **General connection string**, and then locating the parameter `cluster={routing-id}` in your connection string.
```

Comment:
I'm not sure we want to remove this fully. Seems like here is a good place to show the new SNI and say "switch to SNI login for this tool" as a solution.

Reply:
My understanding is also that the routing id still exists, we're just not showing it in the connection string. Maybe we should have some guidance for users who still need it...?